tests/_test_mscolab/test_server.py

```diff
@@ -33,13 +33,19 @@
 from mslib.mscolab.server import check_login, register_user
 from mslib.mscolab.file_manager import FileManager
 from mslib.mscolab.seed import add_user, get_user
+from tests.utils import XML_CONTENT1, XML_CONTENT2
 
 
 class Test_Server:
     @pytest.fixture(autouse=True)
-    def setup(self, mscolab_app):
+    def setup(self, mscolab_app, mscolab_managers):
         self.app = mscolab_app
+        self.sockio, self.cm, self.fm = mscolab_managers
+        self.sm = self.sockio.sm
+        self.sockets = []
         self.userdata = 'UV10@uv10', 'UV10', 'uv10'
+        self.sio = self.sockio.test_client(self.app)
```

Comment:
Please use this as a contextmanager, so it is disconnected properly afterwards.

Reply:
Oh, actually, I don't think it is useable as a contextmanager, unlike the flask test_client. Still, the disconnect is missing.

Reply:
unfortunately context manager protocol is not implemented into the SocketIOTestClient for flask-socketio, but hey I can just use it there where I need it.